### PR TITLE
Version control and update reminder

### PR DESCRIPTION
**Pull request by ezanchi**

### CODE REVIEW

**Comment by fj317** [comment]
What are these changes for generally?

File: PumpBot.py [hunk 1]
```diff
@@ -45,6 +50,25 @@ def log(information):
 manualBTC = float(data['manualBTC'])
 profitMargin = float(data['profitMargin']) / 100
 stopLoss = float(data['stopLoss'])
+currentVersion = float(data['currentVersion'])
+
+# check we have the latest version
+url = 'https://raw.githubusercontent.com/fj317/PumpBot/master/config.json'
+urllib.request.urlretrieve(url, 'version.json')
+try:
```
Comment (fj317) [inline PumpBot.py:58]: This try-catch will always fair since version.json does not exist?

**Comment by fj317** [comment]
I think instead of adding an update reminder, we should focus on creating and releasing our first release of the program in a stable format. Curious on your thoughts on this?

**Comment by ezanchi** [comment]
> I think instead of adding an update reminder, we should focus on creating and releasing our first release of the program in a stable format. Curious on your thoughts on this?

Well I think one goes hand in hand with the other. Since we are doing lots of updates to the code I think it's very useful if the script itself tells the user when there is an update.

We can keep improving the code but this is a very useful feature

File: PumpBot.py (hunk 1 above)
Comment (ezanchi) [reply PumpBot.py:58]: It will only fail if the latest config.json file can not be downloaded and renamed to version.json

Comment (ezanchi) [reply PumpBot.py:58]: The urlretrieve renames the downloaded file into version.json

**Comment by fj317** [comment]
Okay fair enough assuming that works I'll merge it once I'm back at my computer.

**Pull request merged**